bin/newexp-desi

```diff
@@ -38,8 +38,7 @@ parser.add_option("--exptime",  type=int,   help="exposure time in seconds, defa
 parser.add_option("--night",    type=str,   help="YEARMMDD")
 parser.add_option("--nspec",    type=int,   help="Number of spectra to simulate [%default]", default=5000)
 parser.add_option("--airmass",  type=float, help="Airmass [%default]", default=1.0)
-parser.add_option("--randseed", type=int,   help="random number seed"
-                  )
+parser.add_option("--randseed", type=int,   help="random number seed")
```

Comment:
Very minor: Do we want to standardize this variable across scripts to "seed" (as used by the templates) rather than "randseed."